src/main/java/frc/robot/Robot.java

```diff
@@ -260,6 +326,7 @@ public void teleopInit() {
 
   @Override
   public void teleopPeriodic() {
+    // mRobotLogger.info("Angle: " + sGyro.getAngle());
```

Comment:
You could change this to verbose.

Reply:
We don't need it at all anymore. It was there to help debug a specific issue.